openstef/tasks/split_forecast.py

```diff
@@ -69,7 +72,7 @@ def split_forecast_task(
     id pid.
 
     Args:
-        pid (int): Prediction job id
+        pj (int): Prediction job id
```

Comment:
What is used here; the id of the prediction_job, or the complete prediction_job?
If only the id is used, please call the variable `pid`, if the whole prediction_job is used, please change the `(int)` to `PredictionJob`